packages/react-openapi/src/OpenAPISchema.tsx

```diff
@@ -92,6 +93,12 @@ export function OpenAPISchemaProperty(
                             className="openapi-schema-description"
                         />
                     ) : null}
+                    {schema.example ? (
```

Comment:
`example` is not supposed to be markdown so why use markdown here? (https://swagger.io/docs/specification/adding-examples/)

It can result in problems, ex: if an `example` contains a markdown string, it'll be processed there when the user expects to see the raw string.

We want to render a value instead of markdown text.

Reply:
Also the spacing with the description is not great, we should use a gap between the 2

Reply:
This is what it looks like now
<img width="515" alt="Screen Shot 2024-05-15 at 2 20 31 PM" src="https://github.com/GitbookIO/gitbook/assets/140522517/8041a60d-0ecb-4034-907c-184014128e4f">

Reply:
Using
<img width="538" alt="Screen Shot 2024-05-15 at 2 28 24 PM" src="https://github.com/GitbookIO/gitbook/assets/140522517/b585fc19-1eb6-4416-b526-e4e72a97343e">
 margin instead of padding now

Reply:
Also, argos is complaining for the right reasons as Examples are not present in argos. Can I approve it or do I need to update the images argos is using as a baseline? (first time working with argos where errors are justified)

Reply:
If the change is expected and the new version you see in Argos is the correct one, simply approve it and it'll update the baseline.

Reply:
The font styling looks off to me. Have you checked that?

Reply:
The text color seems to stand about a bit too much. Is it black (#000)? Is it the same text color as the property name (`iso-id` in your screenshot)

Reply:
This is what it looks like with the same color as property name

```
.openapi-schema-example {
    @apply mt-2 text-base text-dark/10 dark:text-light/10;
}
```

<img width="1315" alt="Screen Shot 2024-05-20 at 4 50 00 PM" src="https://github.com/GitbookIO/gitbook/assets/140522517/88f2493d-90a5-4721-b1e6-eca5926a776c">

Looks the same to me

Reply:
Idk why but this looks a bit better to me haha. It's subtle but it doesn't come across as fully black. Anyway, lgtm otherwise